mlflow/utils/environment.py

```diff
@@ -500,6 +501,19 @@ def _process_pip_requirements(
     return conda_env, pip_reqs, constraints
 
 
+def _check_for_duplicate_requirements(requirements):
```

Comment:
```suggestion
def _find_duplicate_requirements(requirements):
```

maybe `find` is more straightforward?

Reply:
good point :)